addons/mail/models/mail_activity_template.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class MailActivityTemplate(models.Model):
+    _name = 'mail.activity.template'
+    _description = 'Activity template'
+    _rec_name = 'summary'
+    _check_company_auto = True
```

Comment:
Really ? This is a bit hardcore... we use it generally when having strong constraints linked to the business, not sure we want to raise errors at this level ?